app/models/user.rb

```diff
@@ -8,14 +8,13 @@ class User < ActiveRecord::Base
             format: {
               with: /\b[A-Z0-9._%a-z\-]+@(?:[A-Z0-9a-z\-]+\.)+[A-Za-z]{2,4}\z/
             }, uniqueness: true
-  has_many :listings
   has_many :listing_images, :through => :listings
-  has_many :orders
+  has_many :reservations
   has_attached_file :image, styles: { medium: "300x300>",
                                       thumb: "100x100>" },
                                       default_url: "/images/:style/missing.png"
   validates_attachment_content_type :image, content_type: /\Aimage\/.*\Z/
-
+  scope :host, -> (host) { where()}
```

Comment:
Unused block argument - `host`. If it's necessary, use `_` or `_host` as an argument name to indicate that it won't be used. Also consider using a proc without arguments instead of a lambda if you want it to accept any arguments but don't care about them.<br>Do not use parentheses for method calls with no arguments.<br>Space missing inside }.